apps/vast/pyvast_threatbus/pyvast_threatbus.py

```diff
@@ -385,15 +386,15 @@ async def ingest_vast_ioc(vast_binary: str, vast_endpoint: str, indicator: Indic
     @param vast_endpoint The endpoint of a running vast node ('host:port')
     @param indicator The STIX-2 Indicator to query VAST for
     """
-    global logger
+    global logger, matcher_name
```

Comment:
It's a bit weird to me that the matcher name is a global. Does this mean I cannot set up Threat Bus to feed multiple matchers?

Reply:
Currently yes, that was also the situation before the matcher changes. There is a TODO-comment somewhere in this file to support more fine-grained matcher config, so most likely we can dig out the related clubhouse story for this with some searching.